### ISSUE TITLE
TableView for items in UI

### ISSUE THREAD

**Issue by mirisbowring** [eikek/docspell#923]
It would be nice, if one could switch from Cardview to table view (with custom columns) to see more informations when searching for documents.

e.x:

|  Correspondent | Concerning | Title | Folder | Date Created |
| --- | --- | --- | --- | --- |
| Enterprise AG | Peter Parker | Invoice for Item ABC... |  Peter Private | 25.04.2021 |
| Google Inc. | Bob | Notification about a new co... |  | 25.04.2021 |

With such view, one could visually scan much more documents.

**Comment by eikek** [comment]
Yes, you're right - this is missing! There is already an issue #526 .
